### ISSUE TITLE
Test sorted sets API

### ISSUE THREAD

**Issue by mijicd** [zio/zio-redis#65]
Expand the [ApiSpec](https://github.com/zio/zio-redis/blob/master/src/test/scala/zio/redis/ApiSpec.scala) with sorted sets commands.

**Comment by maciejbak85** [comment]
I will work on it

**Comment by mijicd** [comment]
@maciejbak85 all yours, thanks for taking care of it!

**Comment by maciejbak85** [comment]
While working on it
I found out that `ZINTERSTORE` command does not work correctly.
Probably similar issue is with zRem and zCard I will fix it by the way.

Leaving comment there just in case someone would like to open bug or fix it ;)

**Comment by mijicd** [comment]
@maciejbak85 definitely write test, and check out if it is possible to fix them easily (we had situations with buffer overflows and missing output formats). In case there's a bigger work needed, mark the tests as ignored and document your findings in separate issue. That way we can tackle them in parallel.

**Comment by maciejbak85** [comment]
@mijicd btw Ive fixed them, still finishing those tests, just few left, PR should be ready tomorrow

**Comment by mijicd** [comment]
@maciejbak85 looking forward to reviewing it!
